pymc/gp/cov.py

```diff
@@ -408,6 +526,9 @@ def diag(self, X):
     def full(self, X, Xs=None):
         raise NotImplementedError
 
+    def psd(self, omega):
+        raise NotImplementedError
+
 
 class Periodic(Stationary):
     r"""
```

Comment:
Should the docstrings for the two different kernels be associated with their respective methods?

Reply:
makes sense, moved them down to here